app/views/lecturers/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<p id="notice"><%= notice %></p>
```

Comment:
We dont need views for lecturers. the info for lecturers will be presented with the associated resource, subjects